docs/products/mysql/howto/migrate-database-mysqldump.rst

```diff
@@ -0,0 +1,107 @@
+
+Prerequisites
+-------------
+
+* The ``mysqldump`` `tool <https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html>`_ installed. Check out the `official MySQL <https://dev.mysql.com/doc/mysql-shell/8.0/en/mysql-shell-install.html>`_ documentation on how to install it.
+  
+* A MySQL database that we aim to copy our data from. We will refer to it as ``source-db``.
+  
+* A target MySQL database that we aim to dump our ``source-db`` data. We will refer to it as ``target-db``.
+
+.. tip::
+
+    For the dumping process, we recommend you pick a plan size that is large enough to store your data, so you can limit the downtime during migration.
+
+To simplify the example, we'll create Aiven for MySQL databases for both, ``source-db`` and ``target-db``. You can create the databases by going to your running service for **Aiven for MySQL** > **Databases** > **Create a new database**.
+
+
+Backup the data
+---------------
+
+First, we'll back up our ``source-db`` data to a file called ``mydb_backup.sql``. For that, you need to collect some information about your Aiven for MySQL ``source-db`` database. Go to your Aiven ``source-db`` service, from **Overview** page, find the following information:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Variable
+     - Description
+   * - ``SOURCE_DB_HOST``
+     - **Host** name for the connection
+   * - ``SOURCE_DB_USER``
+     - **User** name for the connection
+   * - ``SOURCE_DB_PORT``
+     - Connection **Port** number
+   * - ``SOURCE_DB_PASSWORD``
+     - Connection **Password**
+   * - ``DEFAULTDB``
+     - Database that contains the ``source-db`` data
+
+Use the following command to backup your Aiven for MySQL data to the ``mydb_backup.sql`` file:
+
+.. code-block:: shell
+
+    mysqldump \
+    -p DEFAULTDB -P SOURCE_DB_PORT \
+    -h SOURCE_DB_HOST --single-transaction \
+    -u SOURCE_DB_USER --set-gtid-purged=OFF \
+    --password > mydb_backup.sql
+
+
+With this command, the password will be requested at the prompt; paste ``SOURCE_DB_PASSWORD`` to the terminal, then a file named ``mydb_backup.sql`` will be created with your backup data.
+
+.. note::
+  
+  Having the prompt request for the password is more secure than including the password straight away in the command. 
+
+The ``--single-transaction`` `flag <https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_single-transaction>`_ starts a transaction before running. This means that the process does not lock the entire database. This allows ``mysqldump`` to read the database in the current state at the time of the transaction which ensures that your data is reliable
```

Comment:
I'm adding a bit more detail here to make the sentence more understandable by the target audience (DBAs)

```suggestion
The ``--single-transaction`` `flag <https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_single-transaction>`_ starts a transaction in isolation mode ``REPEATABLE READ`` before running, allowing ``mysqldump`` to read the database in the current state at the time of the transaction and therefore ensuring consistency of the data.
```